# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1451460517 -3600
#      Wed Dec 30 08:28:37 2015 +0100
# Node ID 9b21039bf63b32373e653f20432d9a2763132c8c
# Parent  ea50940c362ff074840014f5fed0e76976fd1e1e
fix line number when removing a breakpoint via the gui (bug #46779)

* file-editor-tab.cc (handle_margin_clicked): fix line number for removing a breakpoint

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -368,17 +368,17 @@ file_editor_tab::handle_margin_clicked (
           if (markers_mask && (1 << marker::bookmark))
             _edit_area->markerDelete (editor_linenr, marker::bookmark);
           else
             _edit_area->markerAdd (editor_linenr, marker::bookmark);
         }
       else
         {
           if (markers_mask && (1 << marker::breakpoint))
-            handle_request_remove_breakpoint (editor_linenr);
+            handle_request_remove_breakpoint (editor_linenr + 1);
           else
             {
               if (_edit_area->isModified ())
                 message_cannot_breakpoint_changed_file ();
               else
                 handle_request_add_breakpoint (editor_linenr + 1);
             }
         }
